### ISSUE TITLE
As an application owner I want the app to be deployable to Heroku

### ISSUE THREAD

**Issue by eebbesen** [gdcodeclub/nedry#13]
- [x] Get activeadmin working
- [x] Add postgresql to production group

**Comment by lpillman** [comment]
https://nedry.herokuapp.com/
Currently this is setup to automatic deploy when a new push to the master. 

Still need to add postgresql for this to work.

**Comment by eebbesen** [comment]
Nice!

**Comment by lpillman** [comment]
Hey Eric--I think this is done. Some things I'm not sure if we want to do before we complete: 
- Move the pg branch into master
- Get activeadmin working. 

https://nedry.herokuapp.com/

**Comment by eebbesen** [comment]
Yeah, we merged in the pg branch this morning, @gsaarinen and I worked on active_admin, but not totally done with that branch just yet
